ntp-proto/src/packet/v5/mod.rs

```diff
@@ -39,12 +37,12 @@ impl NtpMode {
         self as u8
     }
 
-    #[allow(dead_code)]
+    #[cfg(test)]
```

Comment:
Why is this and the next not used?

Reply:
I've only seen usages in test code, which is why I replaced the `allow(dead_code)` with `cfg(test)`. I think we should probably figure out why there is so much apparently unneeded code in the v5 parts of the codebase, but I would propose we do this in another PR to keep this one somewhat reasonably sized.